token/oauth/oauth2.go

```diff
@@ -40,7 +40,15 @@ type OauthIdentityProvider struct {
 
 // UserProfile represents a user profile fetched from Identity Provider
 type UserProfile struct {
-	Username string
+	Name          string
+	Username      string
+	GivenName     string
+	FamilyName    string
+	Email         string
+	EmailVerified bool
+	Company       string
+	Approved      bool
+	Subject       string
```

Comment:
might have to reconsider these changes..

Reply:
this looks very much like https://github.com/fabric8-services/fabric8-auth/pull/618/files#diff-93bcdce846109a33d22dca161ef464eeR20 indeed